app/(manager)/inventory.tsx

```diff
@@ -61,7 +62,7 @@ export default function InventoryManagementScreen() {
     };
 
     setInventory([...inventory, newItem]);
-    sessionStorage.setItem('ShopInventory', JSON.stringify([...inventory, newItem]));
+    AsyncStorage.setItem('ShopInventory', JSON.stringify([...inventory, newItem]));
```

Comment:
this can't be in the storage, the user can be evil and change this

Reply:
how can he change it ?